<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StereoProjectionGUI</class>
 <widget class="QMainWindow" name="StereoProjectionGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stereo Projection - Team:: Warp64</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="sigma">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>0.5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="run">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>300</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tau">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>0.167</string>
    </property>
   </widget>
   <widget class="QLabel" name="sigmaLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>50</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Sigma</string>
    </property>
   </widget>
   <widget class="QLabel" name="tauLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tau</string>
    </property>
   </widget>
   <widget class="QPushButton" name="leftImage">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Left Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="ntLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nt</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nt">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>16</string>
    </property>
   </widget>
   <widget class="QLabel" name="stepsLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Steps</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="steps">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>400</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rightImage">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Right Image</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leftImagePath">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="rightImagePath">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="mu">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="muLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>50</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Mu</string>
    </property>
   </widget>
   <widget class="QLabel" name="leftImageView">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>251</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string>                  No image loaded</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="rightImageView">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>251</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string>                    No Image Loaded</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="depthImageView">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>80</y>
      <width>251</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string>                   Press Run to get output!</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="implType">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Hover over buttions for more info...</string>
    </property>
    <property name="title">
     <string>Implementation Type</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="impl0">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>3D Grid with Global Memory</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="impl1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>3D Grid with Texture Memory</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="impl2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>3D Grid with Pitch Allocation</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="impl3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>3D Grid with Shared Memory</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStereo_Projection">
    <property name="title">
     <string>Stereo Projection</string>
    </property>
    <addaction name="actionQuit_2"/>
   </widget>
   <addaction name="menuStereo_Projection"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionQuit_2">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
